requirements.txt

```diff
@@ -177,3 +177,4 @@ certifi>=2023.7.22 # not directly required, pinned by Snyk to avoid a vulnerabil
 jwcrypto>=1.4 # not directly required, pinned by Snyk to avoid a vulnerability
 cryptography>=41.0.0 # not directly required, pinned by Snyk to avoid a vulnerability
 aiohttp>=3.9.0 # not directly required, pinned by Snyk to avoid a vulnerability
+nh3
```

Comment:
@t-book please set a fixed version for the library and include it inside the `setup.cfg`

Reply:
Oh yes, thanks @giohappy